projects/valtimo/plugin/src/lib/plugins/zaken-api/components/start-hersteltermijn/start-hersteltermijn-configuration.component.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Component, EventEmitter, Input, OnDestroy, OnInit, Output} from '@angular/core';
+import {FunctionConfigurationComponent} from '../../../../models';
+import {BehaviorSubject, combineLatest, Observable, Subscription, take,} from 'rxjs';
+import {StartHersteltermijnConfig} from '../../models';
+import {PluginTranslatePipe} from '../../../../pipes';
+
+@Component({
+  selector: 'valtimo-start-hersteltermijn-configuration',
+  templateUrl: './start-hersteltermijn-configuration.component.html',
+  styleUrls: ['./start-hersteltermijn-configuration.component.scss'],
+  providers: [PluginTranslatePipe],
+})
+export class StartHersteltermijnConfigurationComponent
+  implements FunctionConfigurationComponent, OnInit, OnDestroy
+{
+  @Input() save$: Observable<void>;
+  @Input() disabled$: Observable<boolean>;
+  @Input() set pluginId(value: string) {
+    this.pluginId$.next(value);
+  }
```

Comment:
Why not just leave it as a simple @Input?

Reply:
I can't change that. It looks like the parent needs it: `FunctionConfigurationComponent`